src/test/java/io/vertx/kafka/client/tests/AdminClientTest.java

```diff
@@ -564,4 +566,37 @@ public void onComplete(Map<org.apache.kafka.common.TopicPartition, OffsetAndMeta
     }));
   }
 
+  @Test
+  public void testListOffsets(TestContext ctx) {
```

Comment:
Can you add a test for asking offsets about a not existing topic?

Reply:
Ok. I'll add one...

Reply:
added.